testing/rstcheck/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Keith Maxwell <keith.maxwell@gmail.com>
+# Maintainer: Keith Maxwell <keith.maxwell@gmail.com>
+pkgname=rstcheck
+_pyname=rstcheck
+pkgver=3.3.1
+pkgrel=0
+pkgdesc="Checks syntax of reStructuredText and code blocks nested within it"
+url="https://github.com/myint/rstcheck"
+arch="noarch"
+license="MIT"
+depends="python3 py3-docutils py3-setuptools"
+checkdepends="bash"
+# the test suite isn't part of the pypi package:
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/myint/$pkgname/archive/v$pkgver.tar.gz
```

Comment:
```suggestion
source="$pkgname-$pkgver.tar.gz::https://github.com/myint/$pkgname/archive/v$pkgver.tar.gz
```

Reply:
Can you point to a rule or documentation that encourages this change please?

This is just a style preference. Unless there's an explicit policy, I am going to ignore the suggestion in order to avoid extra work that is unnecessary in my view.

I can find five examples of exactly the code before the suggested change, with {}s, in main and thirteen in community: 

```
$ git grep 'source="${pkgname}-${pkgver}' main | wc -l
5
$ git grep 'source="${pkgname}-${pkgver}' community | wc -l
13
```